torch/lib/TH/generic/THTensorMath.c

```diff
@@ -2828,19 +2828,59 @@ TENSOR_IMPLEMENT_LOGICAL(le,<=)
 TENSOR_IMPLEMENT_LOGICAL(ge,>=)
 TENSOR_IMPLEMENT_LOGICAL(eq,==)
 TENSOR_IMPLEMENT_LOGICAL(ne,!=)
+                                                     \
```

Comment:
Any other comments? Like, should I put the VECTOR_IMPLEMENT_FUNCTION in THVector?